src/controller/timeline-controller.js

```diff
@@ -233,8 +234,12 @@ class TimelineController extends EventHandler {
     else if (frag.type === 'subtitle') {
       if (payload.byteLength) {
         // We need an initial synchronisation PTS. Store fragments as long as none has arrived.
-        if (typeof this.initPTS === 'undefined') {
+        if (!this.initPTS[frag.cc]) {
```

Comment:
```suggestion
        if (!Number.isFinite(this.initPTS[frag.cc])) {
```

I think an `initPTS` of 0 is valid

Reply:
👍